lib/pure/net.nim

```diff
@@ -812,58 +812,6 @@ proc acceptAddr*(server: Socket, client: var owned(Socket), address: var string,
         let ret = SSLAccept(client.sslHandle)
         socketError(client, ret, false)
 
-when false: #defineSsl:
-  proc acceptAddrSSL*(server: Socket, client: var Socket,
```

Comment:
/cc @dom96 since you moved that code here in 2ce9f1c77f9aa4504c55c75c57e74a5ad840916d